common/scala/src/main/scala/org/apache/openwhisk/http/BasicHttpService.scala

```diff
@@ -156,24 +163,50 @@ trait BasicHttpService extends Directives {
 }
 
 object BasicHttpService {
+  implicit val tid = TransactionId(systemPrefix + "http_service")
+  //start with ready true
+  protected[http] var ready = true
```

Comment:
is this used for anything?

Reply:
The `ready` var? It is used to alter the response to `/ready` endpoint defined in `BasicRasService`.
I'll try to refactor this so that we're not modifying the object var from different places.